api/services/Encryptor.js

```diff
@@ -33,30 +33,34 @@ function encrypt(value, key, { hintSize = 4 } = {}) {
   return { ciphertext, hint };
 }
 
-function encryptObjectValues(obj, key, { hintSize = 4 } = {}) {
-  return _.mapObject(obj, (value) => {
+function encryptObjectValues(
+  obj,
+  encryptionKey,
+  { hintSize = 4, onlyEncryptKeys = [] } = {}
+) {
+  return _.mapObject(obj, (value, key) => {
     if (_.isArray(value) || _.isFunction(value)) {
```

Comment:
I'm not sure that this is part of the issue but at the same time we're here, we should change `_isArray(value)` to return `_.map(v => encryptObjectValues(v, ...)`

Reply:
It's not recursively mapping arrays with an item that is an array. Let me know what you think.